syntax = "proto3";


message TypeThing{
    string sub_name = 1;
    int32 lower_price = 2;
    int32 upper_price = 3;
}

message Answer{
    int32 number_things = 1;  
}

service ThingsSearcher{
    rpc ListClothes(TypeThing) returns (Answer) {};
}